web/Views/Landlord/_RegisterPagePartial.cshtml

```diff
@@ -71,22 +71,20 @@
         <div class="col-1"></div>
         <div class="col-md-5">
             <div class="form-group my-3">
-                <label asp-for="LandlordProvidedCharterStatus" class="form-label">Charter status</label>
-                <select asp-for="LandlordProvidedCharterStatus" class="form-select" required
+                <label for="Select" class="form-label">Charter status</label>
+                <select class="form-select" required
                         id="Select" onclick="checkIfSelectedValueIsTarget('true')">
-                    <option selected disabled value="">Charter Status</option>
-                    <option value="true">Already approved</option>
-                    <option value="false">Not yet approved</option>
+                    <!option @(Model.MembershipId != null ? "selected" : "") value="true">Already approved</!option>
+                    <!option @(Model.MembershipId == null ? "selected" : "") value="false">Not yet approved</!option>

```

Comment:
What do the exclamation marks do?

Reply:
They force it to use regular HTML tags instead of the razor tag helpers. I needed this so I could inject the "selected" attribute into the tag, which the tag helpers don't allow you to do!

Reply:
I would expect `<option selected="@(Model.MembershipId == null ? "true" : "false")">` to work, but fine if not